### PR TITLE
Feature/XIVY-12741 news cron

### PR DESCRIPTION
**Pull request by weissreto**

### CODE REVIEW

File: src/app/pages/news/11.2/06-cron.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 ## CRON expressions {#cron}
 
-Blah cron 
+You can now configure periodical jobs by using <a href="https://en.wikipedia.org/wiki/Cron">CRON</a> expressions.
+CRON expressions give you a wide spectrum of possible configurations. From once every minute up to once every day, week, month, or year. 
+
+- **User Synchronisation**: Define the time the user synchronization is executed with a CRON expression. 
+- **Timer Bean**: Use a CRON expression on the new Start Event Bean `ch.ivyteam.ivy.process.eventstart.beans.TimerBean` to define when your processes are started.
+- **Poller API**: If you write your own Start and Intermediate Event Beans you can use the new <a href="${docBaseUrl}/public-api/ch/ivyteam/ivy/process/beans/IPoller.html">Poller</a> API to define when your Bean should be polled.
```
Comment (ivy-rew) [inline src/app/pages/news/11.2/06-cron.md:8]: I'd skip the 'conditional' writing style to make this sentence easier to read: e.g. ..

Poller API: The definition of polling intervals in custom Start and Intermediate Events has become much easier with the new fluent poll() APIs.

File: src/app/pages/news/11.2/06-cron.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 ## CRON expressions {#cron}
 
-Blah cron 
+You can now configure periodical jobs by using <a href="https://en.wikipedia.org/wiki/Cron">CRON</a> expressions.
+CRON expressions give you a wide spectrum of possible configurations. From once every minute up to once every day, week, month, or year. 
+
+- **User Synchronisation**: Define the time the user synchronization is executed with a CRON expression.
```
Comment (ivy-rew) [inline src/app/pages/news/11.2/06-cron.md:6]: Maybe we want to emphasize, that there was a limitation in the previous version that is now gone?

User Synchronisation: Is now defined by a CRON expression, therefore now also allowing synchronizations multiple times a day or only on weekends.

File: src/web/images/news/11.2/cron/01_user_synch.gif
Comment (ivy-rew) [inline src/web/images/news/11.2/cron/01_user_synch.gif:1]: I think our customers don't like the dark mode due to its bad readability on some devices? I think we better do it all in white.

File: src/web/images/news/11.2/cron/02_timer_bean.png
Comment (ivy-rew) [inline src/web/images/news/11.2/cron/02_timer_bean.png:1]: I'd show this bean configuration on the new inscription editor ... and without grammarly overlays ;-)

**Review by ivy-rew — APPROVED** [review]

File: src/web/images/news/11.2/cron/02_timer_bean.png
Comment (weissreto) [reply src/web/images/news/11.2/cron/02_timer_bean.png:1]: Hmmm, no longer IvyScript editor support on these inscription views?

**Pull request merged**